sway/tree/view.c

```diff
@@ -236,7 +236,12 @@ void view_autoconfigure(struct sway_view *view) {
 		view->border_top = false;
 	}
 
-	switch (view->border) {
+	enum sway_container_border border = view->border;
+	if (view->using_csd) {
```

Comment:
Maybe switching CSD should also change `view->border` and save the previous value in `saved_border`? (I think https://github.com/swaywm/sway/issues/2491 is due to this as well)

Reply:
If we did that then we'd have to change the `border` command so it either ignores the command or writes to `view->saved_border` if CSD is in effect. But on the other hand, at the moment the rendering code has to check the `using_csd` value and omit the borders if so. I'm not sure which is best.